roles/transformers/defaults/main.yml

```diff
@@ -8,6 +8,8 @@ transformers_tengine_environment:
 transformers_ats_keystore: {}
 transformers_include_ghostscript: false
 
+transformers_repository_acs_edition: Enterprise
```

Comment:
to be really pragmatic, both here and in the repo role you can really just have the boolean var